cleanlab/classification.py

```diff
@@ -488,20 +559,26 @@ def find_label_issues(
         <cleanlab.classification.CleanLearning.fit>`. See there for
         documentation for the arguments `pred_probs`, `thresholds`, etc.
 
+        Parameters
+        ----------
+        save_space : bool, optional
+          If True, then returned `label_issues_df` will not be stored as attribute.
+          This means some other methods like `self.get_label_issues()` will no longer work.
+
         Returns
         -------
-        label_issues_mask : np.array
-          A boolean mask for the entire dataset where ``True`` represents a
-          label issue and ``False`` represents an example that is accurately
-          labeled with high confidence.
+        label_issues_df : pd.DataFrame
+          DataFrame with same format as the one returned by :py:meth:`CleanLearning.fit()
+        <cleanlab.classification.CleanLearning.fit>`.
+          See there for documentation regarding column definitions.
         """
 
-        if self.label_issues_mask is not None and self.verbose:
+        if self.label_issues_df is not None and self.verbose and not save_space:
             print(
-                "Overwriting previously identified label issues stored at self.label_issues_mask. "
+                "Overwriting previously identified label issues stored at self.label_issues_df. "
                 "`self.get_label_issues()` will now return the newly identified label issues. "
-                "If you already ran `self.find_label_issues()` and don't want to recompute, you"
-                "should pass the `label_issues_mask` in as a parameter to this function."
+                "If you already ran `self.find_label_issues()` and don't want to recompute, you "
+                "should pass the `label_issues_df` in as a parameter to this function."
```

Comment:
Bug? this method doesn't have `label_issues_df` as a parameter? How can you ask the user to pass it in?

Reply:
good catch, I moved part of the print statement elsewhere in latest commit.